src/controllers/document.controller.js

```diff
@@ -167,15 +167,22 @@ exports.editDocument = async function (req, res, next) {
       return res.status(404).json({ error: "Document Not Found" });
     }
 
-    fields.forEach(({ fieldId, fieldValue }) => {
-      const field = document.fields.id(fieldId);
+    fields.forEach(
+      ({ fieldId, fieldValue, xCoordinate, yCoordinate }, index) => {
+        const field = document.fields.id(fieldId);
 
-      if (!field) {
-        return res.status(404).json({ error: "Field Not Found" });
-      }
+        if (!field) {
+          return res.status(404).json({ error: "Field Not Found" });
+        }
 
-      field.fieldValue = fieldValue;
-    });
+        field.fieldValue = fieldValue;
+
+        database.documents.forEach((eachDocument) => {
+          eachDocument.fields[index].xCoordinate = xCoordinate;
+          eachDocument.fields[index].yCoordinate = yCoordinate;
+        });
+      },
+    );
```

Comment:
```suggestion
        field.fieldValue = fieldValue;

        database.documents.forEach((document) => {
          document.fields[index].xCoordinate = xCoordinate;
          document.fields[index].yCoordinate = yCoordinate;
        });
      },
    );
```

각 다큐먼트도 너무 세세하고 좋지만 그냥 다큐먼트로 해도 충분할거 같은데 혹시 유진님은 어떻게 생각하시나요??

Reply:
네 전체적인 용어사용 통일성도 감안해서 document로 가야하겠네요!! 적용하겠습니다!

Reply:
164번줄에서 document가 이미 선언 되어서 쓰지 못하게 되어서 eachDocument라고 써주었었어요..!

<img width="520" alt="スクリーンショット 2023-07-29 14 49 56" src="https://github.com/Team-Dataface/DataFace-server/assets/83858724/d7c0e42f-d40b-4b38-9db0-0b4b729441e1">

Reply:
아하.. 인자라서 상관 없지 않을까 싶었는데 그것도 lint에서 걸리는군요! 
그러면 명시해주신 eachDocument면 충분히 좋은 네이밍 인것 같습니다!! 👍 👍